README.md

```diff
@@ -2,31 +2,202 @@
+| `p = new Person();` | ![person step2](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step2.png) <br>Parameter <code>p</code> points to the new object.<br>Variable <code>employee</code>still points to the original object due to pass by value.<br>If Java used pass by reference, <code>employee</code>would also point to the new object. |
+| `p.name = "Jie";`   | ![person step3](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step3.png) <br>The name of the new object is changed.                                                                                                                                                                                                    |
+| `p.age = 50;`       | ![person step4](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step4.png) <br>The age of the new object is changed.                                                                                                                                                                                                     |
+
+
+After returning to the `main` method, we see the `age`  has been updated,
+but `employee` still points to the same `Person` object in memory.  If Java
+implemented pass by reference, the `employee` variable would point to the
+50-year-old person named "Jie".
+
+![person step5](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/person_step5.png) 
+
+## Passing an array into a method
+
+
+
+<details>
+    <summary>Can you predict what is printed by the <code>main</code> method for the class below?
+    Step through with the debugger to see how pass by value works with arrays.</summary>
+
+  <p><br>before testArray:[88, 99, 77, 100]<br>after testArray:[10, 99, 77, 100]</p>
+
+</details>
+
+<br>
+
+```java
+import java.util.Arrays;
+
+public class IntArray {
+
+    public static void testArray(int[] nums) {
+        nums[0] = 10;
+        nums = new int[3];
+        nums[0] = 20;
+    }
+
+    public static void main(String[] args) {
+        int[] grades = {88,99,77,100};
 
-## Topic 1
+        System.out.println("before testArray:" + Arrays.toString(grades) );
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+        testArray(grades);
 
-## Topic 2
+        System.out.println("after testArray:" + Arrays.toString(grades) );
+    }
+}
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+```
 
 ## Conclusion
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+Pass by value means the argument value is copied into the parameter variable.
```

Comment:
Maybe reiterate here that Java **only** makes use of pass-by value too

Reply:
ok updated.